card/spec/models/card/trash_spec.rb

```diff
@@ -117,8 +117,8 @@
       @b = Card["B"]
       @a.delete!  #trash
       Rails.logger.info "\n\n~~~~~~~deleted~~~~~~~~\n\n\n"
-      
-      @b.update_attributes! :name=>"A", :update_referencers=>true
+
+      @b.update_attributes! name: "A", update_referencers: true
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.